docs/install.md

```diff
@@ -81,6 +81,9 @@ In the build folder
 #On Linux
 > CXX=g++ cmake -DCMAKE_INSTALL_PREFIX=../install ../src/fringe
 
+# with anaconda (on Linux) 
+> CXX=g++ cmake -DCMAKE_INSTALL_PREFIX=../install ../src/fringe -DCMAKE_PREFIX_PATH=$CONDA_PREFIX
```

Comment:
I would prefer adding the CONDA_PREFIX as a solution just like the GDAL one, in a later section below. I havent had to do this for GDAL / cython on couple of different setups.